test/unit-test/test_usart.c

```diff
@@ -0,0 +1,377 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <cmocka.h>
+
+// including C file to test static functions
+#include <commander/commander.c>
```

Comment:
Is this needed? We haven't typically tested static functions before. The motivation has been that all code should be accessible through public functions.

Reply:
Nope, these are stray includes from `test_hww.h`. I'm pushing a cleanup for these.

Reply:
Fixed.